lib/IDE/IDERequests.cpp

```diff
@@ -392,13 +395,13 @@ void swift::simple_display(llvm::raw_ostream &out, const CursorInfoOwner &owner)
 }
 
 void swift::ide::simple_display(llvm::raw_ostream &out,
-                                const ide::ResolvedCursorInfo &info) {
-  if (info.isInvalid())
+                                ide::ResolvedCursorInfoPtr info) {
```

Comment:
In general I'd prefer things that were being passed a const ref to remain that way (especially for methods like this).